common/crypto/attestation-api/evidence/verify-evidence.cpp

```diff
@@ -187,6 +187,8 @@ bool verify_ias_evidence(
                  extract_hex_from_report(ias_report,
                      offsetof(sgx_quote_t, report_body) + offsetof(sgx_report_body_t, mr_enclave),
                      sizeof(sgx_measurement_t), hex_id));
```

Comment:
should we put here a normalization step (toUpper or to Lower) for both terms?

Reply:
i documented it now in the .md as supposed to be upper-bound.  if we do normalization, i think we would have to do it in peer cli and management API; doing it only here might still bring up issues elsewhere if you confuse cases (e.g., consider org1 installs lowercase, the other one uppercase; from an ERCC case they would now be considered the same but from a ledger perspective they are different!)

Reply:
I understand, but that upper-case/lower-case encoding is irrelevant -- security-wise -- and just raises implementation issues.
We can discuss that, and I can do the change if we decide so.

Reply:
irrelevant security-wise, yes; but functionality-wise it's not as fabric is case-sensitive (at least i thought) for versions.   In that case i don't really see how we could normalize, certainly not transparent to admins? (non-admins should never see the version). But we can talk tomorrow in the call if you want ...